packages/flutter/lib/src/services/hardware_keyboard.dart

```diff
@@ -117,7 +117,7 @@ abstract class KeyEvent with Diagnosticable {
   /// values are not necessarily the same HID codes produced by the hardware and
   /// presented to the driver, because on most platforms Flutter has to map the
   /// platform representation back to an HID code since the original HID
-  /// code is not provided. USB HID is simply a conveniently well-defined
+  /// code is not provided. USB HID is merely a conveniently well-defined
```

Comment:
I went with:
```
  /// Also, even though physical keys are defined with USB HID codes, their 
  /// values are not necessarily the same HID codes produced by the hardware and 
  /// presented to the driver. On most platforms, Flutter has to map the 
  /// platform representation back to an HID code because the original HID 
  /// code is not provided. USB HID was	chosen because it is a well-defined 
  /// standard for referring to keys such as those a Flutter app may encounter. 
```

Reply:
Sounds good, although it should probably be "back to a HID code" instead of "back to an HID code ".

Reply:
I suppose it depends on whether you say "hid" or "H - I - D".

Reply:
fixed, thanks. will land on green.